lib/xander/local_state_query_response.ex

```diff
@@ -1,4 +1,8 @@
 defmodule Xander.LocalStateQueryResponse do
```

Comment:
```suggestion
defmodule Xander.Query.Response do
```

Considering this rename since `Xander.Query` should be the only other module depending on this one. Thoughts @daveminer ?

Reply:
Done ✅